types.go

```diff
@@ -0,0 +1,48 @@
+package kubeconfig
+
+// KubeConfigValue is a struct used to create a kubectl configuration YAML file
```

Comment:
```suggestion
// KubeConfigValue is a struct used to create a kubectl configuration YAML file.
```

Nitty again but godoc should be sentences with the `.`.